src/Illuminate/Database/Eloquent/Casts/Json.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Illuminate\Database\Eloquent\Casts;
+
+class Json
```

Comment:
May we be able to have this in the Support package instead?

Reply:
No. That would imply it affects JSON serialization **everywhere**. Having it here demonstrates it is for **casting only**.